oaknational/oak-ai-lesson-assistant
# feat: updates to download resources page

## Description

- Changes to 'download' page as [specified in Miro](https://miro.com/app/board/uXjVKI-2qEQ=/)
- Change progress to show 10 sections rather than 12 (reducing the first three into `lessonDetails`)
- Copy updates
- **Important:** changes functionality of progress checker to reflect the reality that _1 learning cycle_ is a valid state (so 'Learning cycles' gets a tick in this case)
- Changes a bunch of copy to use **Sentence case**

## Issue(s)

Fixes https://www.notion.so/oaknationalacademy/Update-descriptions-of-each-section-on-the-download-resources-page-164770c7a7fa419c94a8367cf64f3d8f?pvs=4

## How to test

1. Go to https://oak-ai-lesson-assistant-a59flyrtc.vercel-preview.thenational.academy
2. Go to a chat -> downloads
3. You should see new progress sections as in the image below
4. You should see the 10 sections change reflected in the progress bar on the chat too

## Screenshots

How it used to look (delete if n/a):


![Uploading image.png…]()


How it should now look:


<img width="1072" alt="image" src="https://github.com/user-attachments/assets/d064058c-ae9e-411a-a355-73b80a35cfc8">
